src/utils.ts

```diff
@@ -1,10 +1,10 @@
-import { flatten, map, uniq } from 'lodash';
+import { map } from 'lodash';
 import * as URIJS from 'urijs';
 
 import { pathToPointer, pointerToPath } from '@stoplight/json';
 
-export function uniqFlatMap<T>(collection?: T[]) {
-  return uniq(flatten(map(collection, Object.keys)));
+export function mapToKeys<T>(collection?: T[]) {
```

Comment:
Doesn't [mapKeys](https://lodash.com/docs/4.17.15#mapKeys) do what you need?

Reply:
@XVincentX, I'm not sure really. I'm mapping an array of objects to an array of arrays of these objects' keys. Can't find usage for `_.mapKeys` here, but maybe you had something on your mind?

Reply:
Yeah I believe you're right. In any case it's not a big deal.